README.md

```diff
@@ -198,27 +194,115 @@ dfe-analytics-dataform also contains two assertions which will fail and alert yo
 
 If either of these assertions fail, most tables generated by ```dfe-analytics-dataform``` will not be updated. This is essential to ensure that data hidden fields is not copied into non-hidden fields in other tables generated by ```dfe-analytics-dataform```.
 
-### Referential integrity assertions
-Referential integrity means that each value of a foreign key stored in a table matches a value of a primary key in a related table. ```dfe-analytics-dataform``` provides optional Dataform assertions which you may use to monitor referential integrity automatically. If a referential integrity assertion fails, the likely cause is that some data has not been streamed correctly to your ```events``` table by ```dfe-analytics```. A short term fix might be to run a ```dfe-analytics``` backfill on that table. Please report persistent referential integrity assertion failures, however, so that they can be investigated.
-
-To configure a referential integrity assertion for a particular field (key), add parameters to the configuration for that key in your dataSchema so that it looks like:
+#### Hiding fields in tables *not* managed by ```dfe-analytics-dataform```
+```dfe-analytics-dataform``` is only capable of hiding fields in tables that it generates. If data is present in other tables in your BigQuery project, you must ensure that the correct policy tag is attached to these fields.
+
+Google provides [documentation](https://cloud.google.com/dataform/docs/policy-tags) describing how to do this for tables generated by Dataform (but not by ```dfe-analytics-dataform```).
+
+To locate fields in your GCP project which have names which indicate that they may be human-readable Personally Identifiable Information (PII) or GDPR special category data, you may wish to execute the Bash shell script below in the GCP cloud console. This will identify all field names anywhere within your GCP project (except datasets beginning ```dataform``` other than the ```dataform``` dataset itself, because these are likely Dataform development or assertion datasets) which contain the patterns ```email```, ```name```, ```dob```, ```birth```, ```national```, ```insurance```, ```trn```, ```phone```, ```postcode```, ```address```, ```mobile```, ```passport```, ```driv```, ```ethni```, ```religio```, ```union```, ```sex```, ```gender```, ```orientation``` or ```disabilit``` (case insensitive), but do not have a policy tag containing ```hidden``` attached.
+
+<details> <summary>scan_for_possible_unhidden_sensitive_pii_in_field_names.sh</summary>
+
+````bash
+#!/bin/bash
+
```

Comment:
I think the majority of mac users will use zsh rather than bash. I am not sure if it needs stating but there would be a few small updates that would be required to allow this to run if you are using zsh.

Reply:
The trello ticket mentions the script supports an allowlist to exclude certain fields/tables. Is this still a requirement?

Reply:
Thanks @ericaporter . This is designed for use in the GCP cloud console which is a bash shell. I'm anticipating that the majority of users will be analysts without access to a local shell (or a mac!). So I went with bash. Is that OK?

Re. the allow list - the original requirement assumed that we'd be implementing this as a Dataform assertion. But there isn't any way for SQL in an assertion to access data on policy tags. This means that we've ended up implementing as a shell script to use in the GCP cloud console instead, which doesn't provide any version controlled code or a way to share shell scripts between different users. (files in the cloud console persist for a user but aren't shared). So while I could build in some sort of allowlist parameter to the script, I think it would be unlikely to be used?

Reply:
@stevenleggdfe that all makes sense! 👍